src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAnAotApp.cs

```diff
@@ -752,6 +752,31 @@ public void NativeAotStaticLib_only_runs_when_switch_is_enabled(string targetFra
             IsNativeImage(publishedDll).Should().BeTrue();
         }
 
+        [Theory]
+        [InlineData(ToolsetInfo.CurrentTargetFramework, "Static")]
+        [InlineData(ToolsetInfo.CurrentTargetFramework, "Shared")]
+        public void NativeAotLib_errors_out_when_eventpipe_is_enabled(string targetFramework, string libType)
```

Comment:
Since `targetFramework` is always `ToolsetInfo.CurrentTargetFramework`, you can remove it from the theory parameters. Or are you intending to test different target frameworks?